README.md

```diff
@@ -437,6 +437,95 @@ export default withOnyx({
 1. The application uses [`react-navigation`](https://reactnavigation.org/) for navigating between parts of the app.
 1. [Higher Order Components](https://reactjs.org/docs/higher-order-components.html) are used to connect React components to persistent storage via [`react-native-onyx`](https://github.com/Expensify/react-native-onyx).
 
+----
+# HybridApp
+
+Currently, the production Expensify app contains both "Expensify Classic" and "New Expensify". The file structure is as follows:
+
+- 📂 [**App**](https://github.com/Expensify/App)
+    - 📂 [**android**](https://github.com/Expensify/App/tree/main/android): New Expensify Android specific code (not a part of HybridApp native code)
+    - 📂 [**ios**](https://github.com/Expensify/App/tree/main/ios): New Expensify iOS specific code (not a part of HybridApp native code)
+    - 📂 [**src**](https://github.com/Expensify/App/tree/main/src): New Expensify TypeScript logic
+    - 📂 [**Mobile-Expensify**](https://github.com/Expensify/Mobile-Expensify): `git` submodule that is pointed to [Mobile-Expensify](https://github.com/Expensify/Mobile-Expensify)
+        - 📂 [**Android**](https://github.com/Expensify/Mobile-Expensify/tree/main/Android): Expensify Classic Android specific code
+        - 📂 [**iOS**](https://github.com/Expensify/Mobile-Expensify/tree/main/iOS): Expensify Classic iOS specific code
+        - 📂 [**app**](https://github.com/Expensify/Mobile-Expensify/tree/main/app): Expensify Classic JavaScript logic (aka YAPL)
+
+If you have access to [`Mobile-Expensify`](https://github.com/Expensify/Mobile-Expensify), you are eligible to build HybridApp. Otherwise you can still continue working on the standalone `NewDot` application.
```

Comment:
Suggestion, it'd be good to make this clear that you should know if this applies to you:

> You can only build HybridApp if you have been granted access to `Mobile-Expensify`. For most contributors, you will be working on the standalone `NewDot` application.

Reply:
Sure, well put! Updated!